docs/standards.md

```diff
@@ -0,0 +1,167 @@
+      <img src="{{ src }} class="fun-image">
+    </div>
+  ```
+   - Good:
+   ```twig
+    <div class="header">
+      <a href="{{ url }}" class="header__link-fancy">{{ content }}</a>
+      <img src="{{ src }} class="header__image-fun">
+    </div>
+  ```
+#### Pattern definition in `.md`
+PatternLab users can view information about each pattern using the "Pattern Info" feature, whose content is supplied by data provided in a `.md` file.
+
+Please follow the [template](#) for creating `pattern.md` files.
+@todo: add template and update link.
+
+### Checklist:
+- [ ] classification: are the elements classified into patterns in a way that makes intuitive sense and aligns with our classification system? (See "PatternLab Organization" above.)
+- [ ] scalability: is the pattern easily reusable if that is its intent, or is it too specific to be easily included elsewhere? Does the base pattern scale easily with the use of variants?
+- [ ] consistency: is the pattern consistently classified, coded, and conceptualized with the existing patterns?
+- [ ] namespacing: does the pattern code comply with the BEM naming system? Is it intuitively clear by looking at the markup what each element's purpose is?
+- [ ] name descriptiveness: is the pattern named something that is, at a glance, recognizable? Is there an "official" (from the requirements) name for the pattern that should be used instead?
+- [ ] content descriptiveness: is the placeholder content descriptive of the type of content that will go there (e.g. "helper text for input above" for helper text)?
+- [ ] cohesion: was the pattern introduced to address an explicit feature, bug, or change request? Has the work followed the existing process for adding new work?
+- [ ] definition: is the pattern fully documented in its `.md` file? Is it obvious from looking at the pattern what its purpose is and how it should be used?
+
+#### Rules for naming:
+- Pattern names should be vague human-readable descriptions of the component. Generally, we should attempt to use the same plain language that the business would use when that language is not used in a different way elsewhere. (e.g. Don't say refer to a component as a "module" because that word has other meaning in the scope of these projects.)
+- Good: `ama__breadcrumbs`
+- Bad: `list--ordered--wayfinder`
+
+#### Naming pattern variants 
+- Variants of patterns with the same data model should be pseudo patterns. Add docs from code_conventions.md here.
+- Variants should be named “[base item] as [variant]” (e.g. "Header as overlay" or "Button as Secondary Color")
+- Icons should have "icon" in the pattern name. 
+
+### Documenting patterns
+- We will create a template for the markdown files for SG2
+- Variants should have their own MD files that point to the main item
+- All patterns should have the following info in .md: x, y, z
+- base colors have variables printed
+- [PROBLEM] Pattern needs more useful presentation
```

Comment:
All of the "[PROBLEM]" things are just copy/pasted from my old SG Audit spreadsheet. They need to be translated into instructable-sounding things instead, with examples of how to avoid the problem/build the pattern correctly.

I still think it's a good idea to have a checklist somewhere that contains everything from all sections - or a checklist in each section, but haven't decided yet.

Just wanted to give some context on what the [PROBLEM] things meant.